doc/api/icu.md

```diff
@@ -0,0 +1,76 @@
+# icu
+
+    Stability: 1 - Experimental
+
+The `icu` module provides an interface to Unicode and Internationalization
+functionality provided by the ICU4C library.
+
+```js
+const icu = require('icu');
```

Comment:
I remember you mentioning there was a function to tell you if the buffer is one byte or not, or am I misremembering something?